Utilities.cmake

```diff
@@ -53,9 +64,9 @@ function(create_test_executable)
   set(oneValueArgs "")
   set(multiValueArgs "")
   cmake_parse_arguments(create_test_executable "${options}" "${oneValueArgs}" "${multiValueArgs}" ${ARGN})
-
+  
```

Comment:
I don't like adding spaces on empty lines

Reply:
Fair enough. I think that it is my editor that did it.